pages/dkp/konvoy/2.2/image-builder/override-files/default-or-files/index.md

```diff
@@ -12,5 +12,7 @@ Konvoy comes with default override files:
 
 - [FIPS](../../../fips/) override file.
 - [Nvidia](../../../choose-infrastructure/aws/gpu/) override file.
+- Offline override file.
+- Offline for FIPS override file.
 
 You can find these override files in the [Konvoy Image Builder repo](https://github.com/mesosphere/konvoy-image-builder/tree/main/overrides).
```

Comment:
I have some hesitation with us linking to `main` for all of these overrides for a given release. It can be okay for now but we should work out a better way to document these for the customer per-release, as `main` won't always work for all releases.